### PR TITLE
Adds top stats

### PR DESCRIPTION
**Pull request by asbjornh**

### CODE REVIEW

File: app/source/components/top-stats/top-stats.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <h3>Most open issues</h3>
+            <p>
+              <b>{issues.name}</b> with <b>{issues.issues}</b> open issues
+            </p>
+          </div>
+        )}
+
+        {!!users.length && (
+          <div className="top-stats-users">
+            <h3>Most activ people</h3>
```
Comment (Lkleven) [inline app/source/components/top-stats/top-stats.jsx]: typo: 'active'

**Comment by Lkleven** [comment]
Should there be a link to stats page?

**Comment by asbjornh** [comment]
> Should there be a link to stats page?

Yes. See [this PR](https://github.com/Creuna-Oslo/dashboard/pull/40)

**Comment by pellebjerkestrand** [comment]
Looks a little shitty in slim viewports.

screen shot 2019-01-07 at 16 04 07

File: app/source/components/top-stats/top-stats.jsx (hunk 1 above)
Comment (pellebjerkestrand) [reply app/source/components/top-stats/top-stats.jsx]: Fixed it.

**Comment by pellebjerkestrand** [comment]
> Looks a little shitty in slim viewports.

This should be fixed now. I think this can be merged.

**Comment by asbjornh** [comment]
> shitty

😢

**Pull request merged**